src/document.rs

```diff
@@ -372,6 +380,24 @@ impl Document {
                     status.dump_error_logs(&output.data);
                 }
             }
+        } else if options.open {
+            let out_file =
+                output_dir
+                    .join(&profile.name)
+                    .with_extension(match profile.target_type {
+                        BuildTargetType::Pdf => "pdf",
+                    });
+            status.note_highlighted("Opening ", &format!("`{}`", out_file.display()), "");
+            if open::that(&out_file).is_err() {
+                status.report(
+                    MessageKind::Error,
+                    format_args!(
+                        "Failed to open `{}` with system handler",
```

Comment:
Here too, the note text should start with a lower-case letter.

Is it possible to use:

```
if let Err(e) = open::that(&out_file) {
   tt_error!("... text ....", out_file.display(); e);
}
```

?

Reply:
Yep, that looks better, changed.